write/spec/handlers/summary/on_new_day_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+require 'write/lib/summary/on_new_day'
+
+describe Summary::OnNewDay do
+  subject { described_class.new }
+  
+  let(:account) { create(:account) }
+  
+  context "on the first day of the month" do
+    around do |example| 
+      Timecop.freeze(Date.today.beginning_of_month) { example.run } 
+    end
+    
+    expect { subject.call }.to have_enqueued_email(SummaryMailer, :notify)
```

Comment:
I don’t think this is the correct way to proceed. `Summary::OnNewDay`’s job is to publish an event called `Summary::Requested`. We already have a handler for that event (`Summary::OnRequested` at write/lib/summary/on_requested.rb). We could add a spec for that event handler too. `Summary::OnRequested` calls the `Summary` model and saves it, but that doesn’t need to be tested since we already have write/spec/models/summary_spec.rb.